tests/acceptance/modules/Invoices/InvoicesCest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+use Faker\Generator;
+
+class InvoicesCest
```

Comment:
Invoices needs a create function. I think it would be a good idea to have a page class for line items, with createLine, deleteLine, UpdateLine methods. Then we could call these from the invoices.